@here/harp-geoutils/lib/projection/Projection.ts

```diff
@@ -29,8 +29,8 @@ export interface Projection {
     ): Bounds;
 
     /**
-     * Projects a point from geo coordinates (latitude, longitude, altitude) to world coordinates
-     * (x,y,z).
+     * Projects a point from geo coordinates (latitude, longitude, altitude) to
```

Comment:
The "standing order" (see confluence) is to use the line length of 100 characters for comments, and to maintain a block-paragraph-style. This is more readable, and if we omit  this rule, and let everybody choose his favorite way of breaking lines, all hell breaks loose. Literally.

Reply:
Thanks for pointing out, my rewrap comments plugin was configured to 80 chars, apologies.